tests/ui/stable-mir-print/operands.stdout

```diff
@@ -254,6 +254,13 @@ fn closures::{closure#0}(_1: {closure@$DIR/operands.rs:47:5: 47:19}, _2: bool) -
         return;
     }
 }
+fn Ctors::Unit() -> Ctors {
```

Comment:
you'll need to run `x test tests/ui/stable-mir-print --bless` and undo this change to stdout